cypress/e2e/devpool.cy.ts

```diff
@@ -199,7 +199,6 @@ describe("DevPool", () => {
     cy.get('div[id="issues-container"]').children().should("have.length", 2);
     cy.get('[for="priority-bottom"]').click();
     cy.get('div[id="issues-container"]').children().should("have.length", 2);
-    cy.get("#filters-bottom").scrollTo("right");
```

Comment:
Why was this line removed?

Reply:
I think @gentlementlegen removed it because it's an unreliable test. It fails intermittently